### PR TITLE
Various changes & fixes

### PR DESCRIPTION
**Pull request by OcelotWalrus**
What this PR contains:
  * colored text while fighting enemies
    * image

  * made a new class: colors where all colors are stored and make the colors re-usable to all other classes.
  * Also various syntax fix (added spaces between text when needed).

### CODE REVIEW

File: battle.py [hunk 1]
```diff
@@ -51,19 +57,21 @@ def fight(player, item):
                         player["max health"] += 5
                         player["health"] = player["max health"]
                         turn = False
-                    # hold weapon if it is one (currently not working)
+                    # hold weapon if it is one
```
Comment (lumbar527) [inline battle.py:60]: Why are you changing this?

**Review by lumbar527 — APPROVED** [review]
Sure!

**Comment by OcelotWalrus** [comment]
You can merge now @lumbar527 .

File: battle.py [hunk 2]
```diff
@@ -51,19 +65,22 @@ def fight(player, item):
...
                     if item_input in player["inventory"] and item[item_input]["type"] == "Weapon":
-                        player["held item"] = item_input
-                        print("You are now holding a/an ", item_input)
+                        player["held item"] = "Axe"
+                        print("You are now holding an Axe")
```
Comment (lumbar527) [inline battle.py:71]:
```suggestion
                        player["held item"] = item_input
                        print("You are now holding a/an ", player["held item"])
```

**Comment by OcelotWalrus** [comment]
> Why are you resolving my review comments without doing anything? I will merge soon, but I don't want that unrelated change.

I've already commited what you wanted.

**Pull request merged**